frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -29,9 +30,9 @@ const StyledTitle = styled(Card)`
 `
 const HorizontalContent = styled.div`
     display: grid;
-    grid-template-columns: auto auto auto;
+    grid-template-columns: auto auto auto auto;
     align-items: end;
-    gap: 1rem;
+    gap: 2rem;
```

Comment:
Instead of specifying gap we can use: 
```
    justify-content: space-between;
```
to use as much as possible of the space. This will prevent "Completed Tasks" from wrapping in Norwegian

Reply:
Or just remove/reduce the right padding of bottomContent to prevent the wrapping